common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -44,7 +44,10 @@ public class CommonValue {
     public static final String ML_MEMORY_META_INDEX = ".plugins-ml-memory-meta";
     public static final String ML_MEMORY_MESSAGE_INDEX = ".plugins-ml-memory-message";
     public static final String ML_STOP_WORDS_INDEX = ".plugins-ml-stop-words";
+    // index used in 2.19 to track MlTaskBatchUpdate task
     public static final String TASK_POLLING_JOB_INDEX = ".ml_commons_task_polling_job";
+    // index created in 3.0 to track all ml jobs created via job scheduler
+    public static final String ML_JOBS_INDEX = ".plugins-ml-jobs";
```

Comment:
We already have multiple system index. To avoid adding too many system index, can we reuse .plugins-ml-task index ?

Reply:
>I can test this out and get back to you

Any update on this?